spec/ibandit/pseudo_iban_assembler_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+describe Ibandit::PseudoIBANAssembler do
+  subject(:assembler) { described_class.new(local_details) }
+
+  describe '#assemble' do
```

Comment:
Would be tempted to :scissors: this inner context and just merge it into the main one, given that there's only one?